state/protocol/badger/mutator_test.go

```diff
@@ -1701,6 +1700,9 @@ func TestEpochFallbackMode(t *testing.T) {
 			require.NoError(t, err)
 			safetyThreshold := rootSnapshot.Params().EpochCommitSafetyThreshold()
 			require.NoError(t, err)
+			rootProtocolState, err := rootSnapshot.ProtocolState()
+			require.NoError(t, err)
+			epochExtensionViewCount := rootProtocolState.GetEpochExtensionViewCount()
```

Comment:
Please include a sanity check here that this value is at least as large as `EpochCommitSafetyThreshold` (it can be shorter, but I think that is a degenerate edge case, that ideally we should be avoiding. The reason is that it is questionable whether we can guarantee finality on shorter timescales than the `EpochCommitSafetyThreshold`)

Reply:
Will handle it in separate PR: https://github.com/onflow/flow-go/issues/6227